.github/workflows/sweep-experiments.yml

```diff
@@ -77,3 +77,10 @@ jobs:
             ```
             git checkout -t upstream/${{ steps.sweep.outputs.branch }}
             ```
+
+  create-issue-on-error:
+    needs: sweep-experiments
+    if: failure()
+    uses: ampproject/amphtml/.github/workflows/create-issue-on-error.yml@main
+    with:
+      mention: '@ampproject/build-on-duty'
```

Comment:
Similar to my past comment, don't think build on duty has enough context

Reply:
wg-infra then?

Reply:
Switched to wg-infra 🙃